spec/features/ips/add_ip_to_location_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/errors_in_form'
+
+describe 'Add an IP to my account' do
+  context 'and the new location is invalid' do
+    let!(:user) { create(:user) }
```

Comment:
Pretty sure this works without the `!`.

```suggestion
    let(:user) { create(:user) }
```